packages/near-sdk-js/src/cli/cli.ts

```diff
@@ -10,13 +10,14 @@ import { rollup } from "rollup";
 import { Command } from "commander";
 import signal from "signale";
 
-import { executeCommand, validateContract } from "./utils.js";
+import { executeCommand, formatGas, parseNamedArgs, validateContract } from "./utils.js";
 import { runAbiCompilerPlugin } from "./abi.js";
+import { Worker } from "near-workspaces";
 
 const { Signale } = signal;
 const PROJECT_DIR = process.cwd();
 const NEAR_SDK_JS = "node_modules/near-sdk-js";
-const TSC = "node_modules/.bin/tsc";
+const TSC = "node_modules/.bin/tsc";  
```

Comment:
Seems like that the formatting is a little bit off. Could you run the formatter here is an example of that?